test-services/redis/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM redis
```

Comment:
If you aren't going to do anything useful with the image, just run this redis image directly in your tests with `run_container("redis")...` and don't make an unnecessary test-service.

Same thing for Mongo.